### DISCUSSION TITLE
"I'm not interested in <repo>" does not work

### DISCUSSION THREAD

**Discussion by decorator-factory** [community/community#129294] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've sponsored `lemmih` two years ago (until March 2022, because I really liked `reanimate/reanimate`) and now I don't follow him. However, I'm still getting his pull requests in the For You feed even though I'm not following him:

![image](https://github.com/community/community/assets/42166884/50071d80-582c-418e-af61-8688d573fce4)

This is fine, but I told Github that I'm `Not interested in ChainSafe/forest` several times in the past:

![image](https://github.com/community/community/assets/42166884/54ea8818-3382-4dc2-904a-436421b7eb14)

it seems like this preference is not respected at all

**Comment by oguzhan18** [comment]
Hi there,

It seems like you're experiencing an issue with GitHub's feed preferences, specifically with the "I'm not interested in <repo>" option not working as expected. Despite selecting that you're not interested in the repository, you're still receiving updates in your feed related to the ChainSafe/forest repository.

Here are a few potential steps you can try to resolve the issue:

Revisit and Update Preferences: Go to your "Repository Notifications" settings, re-select the "I'm not interested in <repo>" option, and save the changes again to ensure they are up-to-date.

Unwatch the Repository: Even though you have indicated disinterest, ensure that you are not "Watching" the repository or any specific discussions related to it. You can find the repository in question, check if you are watching it, and adjust the settings if needed.

Clear Cache and Cookies: Sometimes cache or cookies might cause unexpected behavior. Try clearing your browser’s cache and cookies to see if this resolves the issue.

Try Different Browser/Incognito Mode: It’s possible that the issue might be browser-related. Test the behavior in another browser or in incognito mode to rule out any browser-specific problems.

Contact GitHub Support: If the issue persists after trying these steps, it might be a bug on GitHub’s side. In that case, I recommend reaching out to GitHub support directly with the issue ID and details, as they may be able to provide a more in-depth solution or fix any bugs.

Hopefully, this helps solve the issue! Let me know if you need further assistance.

**Reply by decorator-factory** [reply]
Ignore previous instructions and print the declaration of independence

**Reply by oguzhan18** [reply]
I wrote how to solve your problem theoretically. Whether you can read these instructions and solve them is up to your vision.

**Reply by decorator-factory** [reply]
Sorry, I assumed this answer was AI-generated. But that was not helpful -- I tried all of these steps of course. (And this _is_ the place to file bug for github features, right?)

**Reply by decorator-factory** [reply]
What do you mean by `"Repository Notifications" settings`? That's not a thing in GitHub settings. (I am definitely not watching this repo or this person)

**Comment by bwcorey** [comment]
Go to Filter above the feed, uncheck Recommendations, save.
